test/karma/test-app/invisible-prehydration-default/karma.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { setupDomTests } from '../util';
+
+describe('invisible-prehydration-default', () => {
+  const { setupDom, tearDownDom, tearDownStylesScripts } = setupDomTests(document);
+
+  afterEach(tearDownDom);
+
+  it('the style element will not be placed in the head', async () => {
+    tearDownStylesScripts();
+    await setupDom('/invisible-prehydration-default/index.html', 1000);
```

Comment:
Double checking, is `1000` the best value here? I'm OK with it if it's the only way to avoid a flaky test, but waiting a full second for a test _if_ we don't need it isn't ideal